front-end/src/components/Post/SecondProposal.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useContext } from 'react';
+import { /* Divider, */ Grid } from 'semantic-ui-react';
+import styled from '@xstyled/styled-components';
+import { ApiPromise } from '@polkadot/api';
+import { InjectedAccountWithMeta } from '@polkadot/extension-inject/types';
+
+import { Form } from '../../ui-components/Form';
+import Button from '../../ui-components/Button';
+import { NotificationContext } from '../../context/NotificationContext';
+import { NotificationStatus } from '../../types';
+
+interface Props {
+	className?: string
+	proposalId?: number | null | undefined
+	api?: ApiPromise,
+	apiReady?: boolean,
+	getLinkedAccount: () => Promise<InjectedAccountWithMeta | undefined>
+}
+
+const SecondProposal = ({ className, api, apiReady, getLinkedAccount, proposalId }: Props) => {
+	const { queueNotification } = useContext(NotificationContext);
+
+	const secondProposal = async () => {
+		if (!api) {
+			return;
+		}
+
+		const linkedAccount = await getLinkedAccount();
+
+		if (!linkedAccount) {
+			return;
+		}
+
+		const second = api.tx.democracy.second(proposalId || 0);
+
+		second.signAndSend(linkedAccount.address, ({ status }) => {
+			if (status.isFinalized) {
+				queueNotification({
+					header: 'Success!',
+					message: `Completed at block hash #${status.asFinalized.toString()}`,
```

Comment:
Showing a hash in a notification was weird to me. We should show the blockNumber if it's there, orherwise nothing. This made the norification overwelming.

Reply:
removed warning. block number not available. showing success message